webapp/src/components/common/DataGrid.tsx

```diff
@@ -241,16 +242,7 @@ function DataGrid(props: DataGridProps) {
         // Prevent selecting the row marker column
         if (newSelection.columns.hasIndex(0)) {
           // TODO find a way to have the rows length to select all the rows like other row markers
-          // setSelection({
```

Comment:
Keep this comments, is linked with the TODO

Reply:
This comment look like a personal note rather than something worth documenting, so it should be removed as it's unused, and adds unnecessary clutter.